activiti-services/activiti-services-query/activiti-services-query-rest/src/main/java/org/activiti/services/query/events/handlers/VariableCreatedEventHandler.java

```diff
@@ -29,13 +34,29 @@
 public class VariableCreatedEventHandler implements QueryEventHandler {
 
     private final VariableRepository variableRepository;
+//    private final TaskRepository taskRepository;
+//    private final ProcessInstanceRepository processInstanceRepository;
+
+    private final EntityManager entityManager;
+
+//    @Autowired
```

Comment:
Please, remove commented code.

Reply:
Fixed in https://github.com/Activiti/Activiti/pull/1464/commits/d8722dda92f703643b94487ffd962d93586e2eb1